app/src/main/java/org/openhdp/hdt/ui/tracking/TrackingViewModel.kt

```diff
@@ -24,43 +30,42 @@ class TrackingViewModel @ViewModelInject constructor(
 
     private var job: Job? = null
 
-    fun midnight() = Date().apply {
-        seconds = 0
-        minutes = 0
-        hours = 0
+    fun initialize() {
+        viewModelScope.launch(Dispatchers.Main) {
+            runCatching {
+                val stopwatches = stopwatchRepository.stopwatchDAO.getAllStopwatchesInOrder()
+                val categories = categoryDAO.getAllCategoriesOrdered()
+                stopwatches.mapNotNull { it.asTrackingItem(categories) }
```

Comment:
Maybe that's why only one timer from category is showing.

Reply:
no, I found solution, fixes are available at 
https://github.com/Open-Health-Data-Project/hdp-android/pull/39